assets/stylesheets/style.scss

```diff
@@ -1,2 +1,67 @@
 @import '_govuk-elements';
 
+
+$space_size: 10px;
+
+.box {
+  display: block;
+  border: 1px solid $panel-colour;
+  padding: (18px $gutter $gutter-half $gutter*1.8);
+
+  margin-bottom: $space_size;
+}
+
+.box-fixed-height {
+  min-height: 100px;
+}
+
+.box-link {
+  text-decoration: none;
+  cursor: pointer;
+
+  h1, h2, h3, h4, h5 {
+    color: $link-colour;
+  }
+
+  &:hover {
+    border-color: $black;
+  }
+}
+
+.space-top-double {
+  margin-top: $space_size * 2;
+}
+
+.column-three-quarter {
+  @include grid-column(3 / 4);
+}
+
+.center {
+  text-align: center;
+}
+
+.color-text {
+  color: $text-colour;
+}
+
+.button-secondary{
+  @include button($grey-3);
+}
+
+.action-buttons {
+  input.button {
+    margin-right: 15px;
+    margin-top: 10px;
+  }
+}
+
+.box-min-padding {
+  padding: (18px $gutter-half $gutter-half $gutter-half);
+}
+
+#validation-errors {
+  color: red;
+  margin-bottom: $space_size;
+}
+
+
```

Comment:
can we group these so that related styles are together? will make splitting stuff into separate files (when we eventually need to) easier